apps/playground/app/sink/page.tsx

```diff
@@ -401,9 +401,9 @@ export default function Sink() {
                                               <td key={variant}>
                                                 <DropdownMenu.Root>
                                                   <DropdownMenu.Trigger>
-                                                    <Button variant="soft" color="gray">
+                                                    <IconButton variant="soft" color="gray">
                                                       <DotsHorizontalIcon />
```

Comment:
@vladmoroz  did you mean to update this to `<DropdownMenu.TriggerIcon />` too?

Reply:
Nope, not really, you don't _have_ to use the icon part